### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by christophermluna**
Add mention of 3rd-party tool Geoip2-update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -258,6 +258,18 @@ print($record->network . "\n"); // '128.101.101.101/32'
...
 
+## Database Updates ##
+
+You can keep your databases up to date with our
+[GeoIP Update program](https://github.com/maxmind/geoipupdate/releases).
+[Learn more about GeoIP Update on our developer's
+site.](https://dev.maxmind.com/geoip/updating-databases?lang=en)
+
+There is also a third-party tool for updating databases using PHP and
+composer. MaxMind does not offer support for this tool or maintain it.
```
Comment (oschwald) [inline README.md:269]: "Composer" should be capitalized unless you are referring to the command, in which case it should be surrounded by backtickers.

File: README.md [hunk 2]
```diff
@@ -258,6 +258,18 @@ print($record->network . "\n"); // '128.101.101.101/32'
...
+[Learn more about the Geoip2-update tool for PHP and composer on its
```
Comment (oschwald) [inline README.md:270]: Their GitHub project refers to the tool as "Geoip2 Update".

File: README.md [hunk 3]
```diff
@@ -258,6 +258,18 @@ print($record->network . "\n"); // '128.101.101.101/32'
...
+github page.](https://github.com/tronovav/geoip2-update)
```
Comment (oschwald) [inline README.md:271]: "GitHub"

File: README.md [hunk 4]
```diff
@@ -258,6 +258,18 @@ print($record->network . "\n"); // '128.101.101.101/32'
 
 ```
 
+## Database Updates ##
+
+You can keep your databases up to date with our
+[GeoIP Update program](https://github.com/maxmind/geoipupdate/releases).
+[Learn more about GeoIP Update on our developer's
```
Comment (oschwald) [inline README.md:265]: Maybe "developer portal" instead as that is what the site refers to itself as? The primary reason I am commented is I wasn't sure if this should be singular or plural or possessive or not.

**Review by oschwald — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -262,13 +262,13 @@ print($record->network . "\n"); // '128.101.101.101/32'
...
-site.](https://dev.maxmind.com/geoip/updating-databases?lang=en)
+[Learn more about GeoIP Update on our developer
+portal.](https://dev.maxmind.com/geoip/updating-databases?lang=en)
 
 There is also a third-party tool for updating databases using PHP and
-composer. MaxMind does not offer support for this tool or maintain it.
-[Learn more about the Geoip2-update tool for PHP and composer on its
-github page.](https://github.com/tronovav/geoip2-update)
+Composer. MaxMind does not offer support for this tool or maintain it.
+[Learn more about the Geoip2 Update tool for PHP and composer on its
```
Comment (oschwald) [inline README.md:270]: "Composer".

**Review by oschwald — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (christophermluna) [reply README.md:270]: sorry

**Review by oschwald — APPROVED** [review]

**Pull request merged**